src/client/debug/main/Main.svelte

```diff
@@ -1,6 +1,8 @@
 <script>
   export let client;
 
+  import JSONTree from 'svelte-json-tree';
```

Comment:
If you take a look at [the `dist` code from `svelte-json-tree`](https://unpkg.com/browse/svelte-json-tree@0.1.0/dist/index.js), it’s a CommonJS module (little hard to see on the minimised output, but prettify it and scroll to the bottom and you’ll find `module.exports = Root` with `Root` being the entry component).

I think normally when the Svelte compiler is run on the Debug panel, it resolves `svelte-json-tree` by looking at its `package.json` and following the `svelte` field there (which points to `src/Root.svelte`). All good. However, my suspicion is that under Jest for some reason `svelte-json-tree` gets resolved as a regular non-Svelte module, so we now hit `dist/index.js`, but because that’s a CommonJS module and not an ES module, the import returns `undefined` and throws a “not a constructor” error when the code tries to instantiate `JSONTree`.

So, a solution:

```suggestion
  const JSONTree = require('svelte-json-tree');
```

I think this assures that we import the CommonJS module everywhere (local tests, build & examples work). I don’t know if this is the *ideal* fix, because perhaps depending on this results in a less optimised bundle than compiling the source Svelte component (I guess there’s some boilerplate shared code in the CommonJS transpiled version).

@nicolodavis Do you have any thoughts here? Is there a way we can get Jest to do the right thing in terms of resolving this import?

Reply:
Hmm, I hadn’t noticed that plugin, but it doesn’t seem to be possible to disable it just for a certain file or import. (I’m also not 100% sure if it comes into play. `Main.svelte` is presumably handled by `jest-transform-svelte`, which already outputs CJS modules, so perhaps the import transformation is happening there?) For now it seems like the `require` fix is acceptable. If there is some minimal bump in bundle size, this is only going to be in the development environment in any case.